modules/bootstrap/variables.tf

```diff
@@ -20,3 +20,9 @@ variable "location" {
   description = "Location in which the GCS Bucket will be deployed. Available locations can be found under https://cloud.google.com/storage/docs/locations."
   type        = string
 }
+
+variable "bootstrap_files" {
+  description = "Location to the bootstrap file structure. More infor can be found under https://docs.paloaltonetworks.com/vm-series/9-1/vm-series-deployment/bootstrap-the-vm-series-firewall/bootstrap-package"
+  type        = string
+  default     = "./bootstrap_files"
```

Comment:
```suggestion
```

Reply:
It make sense to have this explicitly specified by the user to avoid any ambiguous outcomes